scripts/us_bjs/nps/nps_statvar_writer.py

```diff
@@ -0,0 +1,156 @@
+NO_GENDER = "Both"
+POPULATIONTYPE_PERSON = "Person"
+POPULATIONTYPE_DEATH = "MortalityEvent"
+POPULATIONTYPE_INCARCERATION = "IncarcerationEvent"
+GENDERS = ["Female", "Male", "Both"]
+RACES = ["WhiteAlone", "BlackOrAfricanAmericanAlone", "HispanicOrLatino", "AmericanIndianOrAlaskaNativeAlone", "AsianAlone","NativeHawaiianOrOtherPacificIslanderAlone", "TwoOrMoreRaces"]
+CAUSES_OF_DEATH = ["JudicialExecution", "IllnessOrNaturalCause", "AIDS", "IntentionalSelf-Harm(Suicide)", "Accidents(UnintentionalInjuries)", "DeathDueToAnotherPerson", "Assault(Homicide)", "NPSOtherCauseOfDeath"]
+INCARCERATION_EVENTS = ["AdmittedToPrison", "ReleasedFromPrison"]
+SENTENCED_OVER_1_YEAR = "MaxSentenceGreaterThan1Year"
+SENTENCED_UNDER_1_YEAR = "MaxSentence1YearOrLess"
+UNSENTENCED = "Unsentenced"
+SENTENCE_AMOUNTS = [SENTENCED_OVER_1_YEAR, SENTENCED_UNDER_1_YEAR, UNSENTENCED]
+JURISDICTION = "MeasuredBasedOnJurisdiction"
+CUSTODY = "MeasuredBasedOnCustody"
+    
+def get_constant_properties(gender, population_type, measurement_qualifier):
+    """gets list of properties needed for all stat vars
+
+    Args:
+        gender: gender of stat var as string
+        population_type: population type of stat var as string
+        measurement_qualifier: measurement qualifier of stat var as string
+    
+    Returns:
+        list of strings where each string is an individual property
+    """
+    props = []
+    if not gender == NO_GENDER:
+        props.append("gender: schema:" + gender + "\n")
```

Comment:
nit: maybe drop all the "\n" at the end of each line and finally '\n'.join(...)?